mteb/tasks/Retrieval/XPQAESRetrieval.py

```diff
@@ -0,0 +1,56 @@
+from datasets import load_dataset
+
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+_EVAL_SPLIT = "test"
+
+
+class XPQAESRetrieval(AbsTaskRetrieval):
+    @property
+    def description(self):
+        return {
+            "name": "XPQAESRetrieval",
+            "hf_hub_name": "jinaai/xpqa",
+            "reference": "https://arxiv.org/abs/2305.09249",
+            "description": ("xPQA is a large-scale annotated cross-lingual Product QA dataset."),
+            "type": "Retrieval",
+            "category": "s2s",
+            "eval_splits": ["test"],
+            "eval_langs": ["es"],
```

Comment:
Can we do all of `["ar", "de", "es", "fr", "hi", "it", "ja", "ko", "pl", "pt", "ta", "zh"]` as you have already uploaded them I think?

Reply:
Ah yes, I think until quite recently it was not possible to create multlingual retrieval tasks, so that is why we implemented just the Spanish version(s) here. I will definitely make it multlingual :) same for the XMarket and Mintaka retrieval tasks above.